server/routes/api/representatives.js

```diff
@@ -89,114 +90,169 @@ router.get('/:searchText', async (req, res) => {
-          (Array.isArray(rep.addresses) && rep.addresses[0]) || {}
-
-        const repInfo = {
-          id: rep.sk || rep.id || '',
-          name: formatName(rep),
-          title:
-            rep.office.title ||
-            rep.office.chamber.name_formal ||
-            rep.office.chamber.name ||
-            '',
-          address_line1: mainAddr.address_1 || '',
-          address_line2:
-            mainAddr.address_2 +
-              (mainAddr.address_3 ? ', ' + mainAddr.address_3 : '') || '',
-          address_city: mainAddr.city || '',
-          address_state: mainAddr.state || '',
-          address_zip: mainAddr.postal_code || '',
-          address_country: 'US',
-          email:
-            (Array.isArray(rep.email_addresses) && rep.email_addresses[0]) ||
-            'Not Made Public',
-          contactPage:
-            rep.web_form_url || (Array.isArray(rep.urls) && rep.urls[0]) || '',
-          photoUrl:
-            rep.photo_origin_url ||
-            'https://cdn.pixabay.com/photo/2016/08/08/09/17/avatar-1577909_1280.png',
-          socialMediaPages: getOfficialSocialMediaPages(rep.identifiers),
-          photoCroppingCSS: getPhotoCroppingValues(rep.photo_cropping)
+  let response
+
+  if (error) {
+    // http://127.0.0.1:8080/api/representatives/91765 get request failed
+    // uncomment only when debugging
+    // console.error(error)
+
+    // to avoid server crashing so front-end work can continue, add NODE_ENV='development' to your .env
+    if (process.env.NODE_ENV === 'development') {
+      // a test_response helpful for front-end work when axios requests are not working
+      const test_response = {
+        errors: [],
+        results: {
+          candidates: [
+            {
+              officials: [
+                // rep:
+                {
+                  office: {
+                    district: {
+                      ocd_id: 1234
+                    },
+                    title: 'Office Title'
+                  },
+                  chamber: {
+                    name_formal: 'Chamber Formal Name',
+                    name: 'Name of Chamber'
+                  },
+                  addresses: [
+                    // mainAddr:
+                    {
+                      address_1: '123 Main',
+                      address_2: 'Apt 321',
+                      city: 'City',
+                      postal_code: '00000',
+                      state: 'California'
+                    }
+                  ],
+                  web_form_url: 'https://fake_web_form_url.com',
+                  email_addresses: 'fake_email@failed_request.com',
+                  photo_origin_url: 'https://fake_photo_origin_url.com',
+                  identifiers: [],
+                  photo_cropping: {}
+                }
+              ]
+            }
+          ]
         }
+      }
 
-        return repInfo
+      response = test_response
+    } else {
+      res.status(500).send({
```

Comment:
Now looking at this, I think line 181 warrants a `return` before the `res.status(500).send`. This is probably why I was seeing `Error: Can't set headers after they are sent to the client` at some point.